docs/notebooks/JAX_for_the_impatient.ipynb

```diff
@@ -0,0 +1,1293 @@
+{
```

Comment:
<p>Maybe show what normal_pseudorandom you'd get for the original key as well? (And you can show a second line showcasing how when you use <code>key</code>  twice you get the same normal random?)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/flax/pull/542/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>Added the normal_pseudorandom for the 3 keys (old, key, subkey). Also added one cell showing that multiple calls with same key gives same output.</p>